D3D12HelloWorld/Games/ProfilingEvent.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Serilog;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Vortice.Direct3D12;
+using wired.Graphics;
+
+namespace wired.Games {
+    /// <summary>
+    /// Inspired by PIX Performance Tuning and Debugging, namely, PIXBeginEvent and PIXEndEvent
+    /// </summary>
+    class ProfilingEvent : IDisposable {
```

Comment:
Straight copy of the class created for D3D12Bundles to port the PIX event behaviour, just updated the namespace.  Seems a useful type outside that sample.